contribution.md

```diff
@@ -0,0 +1,237 @@
+
+If appropriate demo or sample are absent, you must provide your own demo (C++ or Python). Demo's input requires next options:
+```
+    -i "<path>"              Optional. Path to a input file or directory (for multiple inferences). By default input must be generated randomly.
+    -m "<path>"              Required. Path to an .xml file with a trained model
+    -d "<device>"            Optional. Target device for model inference. Usually CPU and GPU. By default CPU
+    -no_show                 Optional. Do not show inference result.
+```
+Also you can add all necessary parameters for inference your model.
+
+## Accuracy validation
+
+To run accuracy validation, use [Accuracy Checker](./tools/accuracy_checker/README.md) tool, provided with repository. Doing this very simple if model task from supported. You must only create accuracy validation configuration file in this case. 
+
+When the configuration file is ready, you must run Accuracy Checker to obtain metric results. If they match your results, that means  conversion was fully successful and Accuracy Checker fully supports your model, metric and dataset. If no - recheck [conversion][#model-conversion] parameters or validation configuration.
+
+*After this step you will get accuracy validation configuration file - **<model_name>.yml***
+
+## Configuration file
+
+Models configuration file contains information about model: what it is, how to download it and how to convert it to IR format. This information must be specified in `model.yml` file, which must be in the models subfolder. Let look closer to the file content.
+
+**`description`**
+
+This tag must contain description of model.
+
+**`task_type`**
+
+This tag describes on of the task that model solves:
+- `action_recognition`
+- `classification`
+- `detection`
+- `face_recognition`
+- `head_pose_estimation`
+- `human_pose_estimation`
+- `image_processing`
+- `instance_segmentation`
+- `object_attributes`
+- `optical_character_recognition`
+- `semantic_segmentation`
+
+If your model solves another task, you can freely add it with modification of [tools/downloader/common.py](tools/downloader/common.py) file list `KNOWN_TASK_TYPES`
+
+**`files`**
+
+You must describe all files, which must be downloaded, in this section. Each file must is described in few tags:
+
+* `name` sets file name after downloading
+* `size` sets file size
+* `sha256` sets file hash sum
+* `source` sets direct link to file *OR* describes file access parameters
+
+If file is located on GoogleDrive\*, section `source` must contain:
+```
+ - $type: google_drive
+   id: <file id>
+```
+**`postprocessing`** (*optional*)
+
+Sometimes right after downloading model are not ready for conversion, or conversion may be incorrect or failure. It may be avoided by some manipulation with original files, such as unpacking, replacing or deleting some part of file. This manipulation must be described in this section.
```

Comment:
Sometimes right after downloading model is not ready for conversion by Model Optimizer and some additional pre processing needed.